.github/workflows/cd.yaml

```diff
@@ -90,32 +90,30 @@ jobs:
           name: ${{ env.OUTPUT }}
           path: ${{ env.OUTPUT }}
 
-  cd_macos_x86_64:
+  cd_macos_x64:
     name: MacOS
-    runs-on: macos-latest
```

Comment:
why not on latest?

Reply:
So that when GitHub updates the latest version we don't get two different caches (because OS updates is done is waves).